Containerfile

```diff
@@ -95,12 +95,13 @@ RUN systemctl disable pmlogger.service
 
 RUN /tmp/workarounds.sh
 
-# Clean up repos, everything is on the image so we don't need them
+# Clean up repos, everything is on the image so we don't need them. Also link cjk fonts to hardcoded steam paths.
 RUN rm -f /etc/yum.repos.d/terra.repo
 RUN rm -f /etc/yum.repos.d/ganto-lxc4-fedora-"${FEDORA_MAJOR_VERSION}".repo
 RUN rm -f /etc/yum.repos.d/vscode.repo
 RUN rm -f /etc/yum.repos.d/_copr:copr.fedorainfracloud.org:phracek:PyCharm.repo
 RUN rm -f /etc/yum.repos.d/fedora-cisco-openh264.repo
+RUN ln -s "/usr/share/fonts/google-noto-sans-cjk-fonts" "/usr/share/fonts/noto-cjk"
```

Comment:
will do so, thanks for checking

Reply:
I would fiddle it between line 35 and 36 and then also remove the old comment you made. Your new location is still only for `bluefin-dx` builds. If you put them there it will roll down in all the `bluefin-*` builds.

Reply:
done. does that look better now?

Reply:
I am not seeing your new version yet

Reply:
sorry, it should be there now.

Reply:
sweet, looking good!